dashboard/config/scripts_json/2016_sciencePD-phase2b.script_json

```diff
@@ -0,0 +1,763 @@
+{
+  "script": {
+    "name": "2016_sciencePD-phase2b",
+    "wrapup_video_id": null,
+    "hidden": true,
+    "login_required": true,
+    "properties": {
+      "professional_learning_course": "CS in Science Legacy"
+    },
+    "new_name": null,
+    "family_name": null,
+    "seeding_key": {
+      "script.name": "2016_sciencePD-phase2b"
+    }
+  },
+  "lesson_groups": [
+    {
+      "key": "",
+      "user_facing": false,
+      "position": 1,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson_group.key": "",
+        "script.name": "2016_sciencePD-phase2b"
+      }
+    }
+  ],
+  "lessons": [
+    {
+      "key": "Welcome!",
+      "name": "Welcome!",
+      "absolute_position": 1,
+      "lockable": false,
+      "relative_position": 1,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "Welcome!",
+        "lesson_group.key": "",
+        "script.name": "2016_sciencePD-phase2b"
```

Comment:
I know this is essentially commenting on the previous PR which implemented this... but I like the way these fields are spelled out within the seeding_key. Looks great for readability / debuggability.